schedule/kernel/ibtest-master.yaml

```diff
@@ -2,6 +2,9 @@ name:           ibtest-master
 description:    >
     The master node for the infiniband testsuite (hpc-testing)
 vars:
+    AGAMA: 1
+    AGAMA_PRODUCT_ID: SLES
+    INST_AUTO: yam/agama/auto/sles_default.jsonnet
```

Comment:
yam folder is dangerous for us to use. Yam team has it as development and they will break stuff. We should use `agama_auto` folder,